### PR TITLE
ROOT 6-22 - get one commit

### PR DESCRIPTION
**Pull request by mrodozov**
6-22 remains with one commit removed

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

**Comment by smuzaffar** [comment]
please test
lets restart based on latest IB

**Comment by mrodozov** [comment]
+externals

**Comment by smuzaffar** [comment]
Is there any progress or the removed commit?

**Comment by mrodozov** [comment]
this is the original issue that was adding the commit 
https://github.com/root-project/root/issues/7754
looking into 6-22 branch it's only one commit more since when we first removed the commit which is not linked to that issue.

File: root.spec [hunk 1]
```diff
@@ -3,8 +3,8 @@
 ## INITENV +PATH PYTHON3PATH %{i}/lib
 ## INITENV SET ROOTSYS %{i}
 
-%define tag d9ead8a7d0a08238c33e4920ff0584097904981b
-%define branch cms/v6-22-00-patches/0c7eac6
+%define tag ae63a2f7f3ad0ec5b3dcd932178b7144ced7786c
+%define branch cms/v6-22-00-patches/c25f7e5
```
Comment (smuzaffar) [inline root.spec]: @mrodozov  what is this branch cms/v6-22-00-patches/c25f7e5 ? c25f7e5 should be a commit from https://github.com/root-project/root/commits/v6-22-00-patches but I do not see it there

**Comment by smuzaffar** [comment]
hold

File: root.spec (hunk 1 above)
Comment (smuzaffar) [reply root.spec]: @mrodozov , please use a valid commit from root branch otherwise automatic update tool will break

Comment (mrodozov) [reply root.spec]: it's fixed to be auto updated

**Comment by mrodozov** [comment]
please test

**Comment by mrodozov** [comment]
+externals

**Pull request merged**